src/main/java/lotto/model/LottoResult.java

```diff
@@ -16,13 +16,27 @@ public LottoResult() {
         matchCounts.put(LottoWinningPrice.FOUR, 0);
         matchCounts.put(LottoWinningPrice.FIVE, 0);
         matchCounts.put(LottoWinningPrice.SIX, 0);
+        matchCounts.put(LottoWinningPrice.BONUS, 0);
     }
 
-    public void addMatchCounts(int matchCount){
+    public void addMatchCounts(int matchCount, boolean isBonus){
         LottoWinningPrice lottoWinningPrice = LottoWinningPrice.getLottoWinningPrice(matchCount);
+        if(!lottoWinningPrice.isView())
+            return;
```

Comment:
조건문 사용시 if 와 괄호 사이에 공백을 넣고
라인이 하나여도 괄호로 감싸주는 것이 컨벤션 입니다.

7.4 if, if-else, if else-if else 문
[[코딩규칙] 자바 코딩 규칙(Java Code Conventions)](https://myeonguni.tistory.com/1596)

```suggestion
        if (!lottoWinningPrice.isView()) {
            return;
        }
```